src/renderer/corporate/member/member-shares-form.jsx

```diff
@@ -133,6 +135,7 @@ const CorporateMemberSharesFormComponent = withStyles(styles)(props => {
 							required
 							error={errors.equity}
 							value={equity}
+							disabled={!positions.some(p => positionsWithEquity.includes(p))}
 							onChange={onFieldChange('equity')}
```

Comment:
LGTM

Reply:
That line is difficult to grasp.
It's probably better to have an helper function for that.